src/styles/_catalog.scss

```diff
@@ -1,22 +1,16 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 200px;
```

Comment:
what are 200px? it would be more understandable when we use this like variables named like 'card-width' so we understand what this size is

Reply:
Agree, please fix it